java/openmldb-batch/pom.xml

```diff
@@ -438,6 +438,22 @@
 			<plugin>
 				<groupId>pl.project13.maven</groupId>
 				<artifactId>git-commit-id-plugin</artifactId>
+				<version>2.2.4</version>
```

Comment:
there is a parent config in `java/pom.xml`, is that configuration not right ?
If so, we may tweak options there instead, so all modules shared the same config

Reply:
The parent plugin did not generated version file and we can not load the version. I will check the plugin config again and reuse the parent if possible.